app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for person in people:
+        person_list.append(Person(person["name"], person["age"]))
+
+    for person_dict in people:
+
+        spouse_key = "husband" if person_dict.get("husband") is not None \
+            else "wife"
+        spouse_name = person_dict.get(spouse_key)
```

Comment:
The `get` method will return `None` if the `spouse_key` is not present in the dictionary. Ensure that this is the intended behavior and handle cases where `spouse_name` might be `None`.